sspl_test/sspl-test.spec

```diff
@@ -27,7 +27,7 @@ URL:        https://github.com/Seagate/cortx-sspl
 Source0:    %{name}-%{version}.tgz
 Requires:   %{product_family}-sspl = %{version}-%{release}
 BuildRoot:  %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
-Requires:  python36-psutil
+Requires:  (python36-psutil or python3-psutil)
```

Comment:
Ok. functionally tested with new package python3-psutil righ ?

Reply:
yes tested it.. works on both centos and rocky.. 👍